Tests/SwiftParserTest/DiagnosticTests.swift

```diff
@@ -65,6 +70,11 @@ public class DiagnosticTests: XCTestCase {
       Syntax(raw: $0.parseFunctionSignature().raw).as(FunctionSignatureSyntax.self)!
     }
 
-    XCTAssertSingleDiagnostic(in: signature, line: 1, column: 7, message: "'throws' may only occur before '->'")
+    XCTAssertSingleDiagnostic(
+      in: signature,
+      line: 1, column: 7,
```

Comment:
It'd be nice to add a marker equivalent to the structure tests. Maybe not as important here, I just highly dislike referencing line/col in tests in general 😅